amd_openvx_extensions/amd_rpp/source/Resizetensor.cpp

```diff
@@ -0,0 +1,375 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+
+struct ResizetensorLocalData
+{
+    RPPCommonHandle handle;
+    rppHandle_t rppHandle;
+    Rpp32u device_type;
+    Rpp32u nbatchSize;
+    RppiSize *srcDimensions;
+    RppiSize maxSrcDimensions;
+    RppiSize *dstDimensions;
+    RppiSize maxDstDimensions;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    Rpp32u *srcBatch_width;
+    Rpp32u *srcBatch_height;
+    Rpp32u *dstBatch_width;
+    Rpp32u *dstBatch_height;
+    RpptDescPtr srcDescPtr, dstDescPtr;
+    RpptROIPtr roiTensorPtrSrc;
+    RpptRoiType roiType;
+    RpptImagePatchPtr dstImgSize;
+    RpptDesc srcDesc, dstDesc;
+#if ENABLE_OPENCL
+    cl_mem cl_pSrc;
+    cl_mem cl_pDst;
+#elif ENABLE_HIP
+    void *hip_pSrc;
+    void *hip_pDst;
+    RpptImagePatch *d_dstImgSize;
+    RpptROI *d_roiTensorPtrSrc;
```

Comment:
Instead of d_* we can probably change the variable names to have hip_ to maintain uniformity